repair/repair.cc

```diff
@@ -1015,6 +1017,15 @@ int repair_service::do_repair_start(sstring keyspace, std::unordered_map<sstring
     auto id = repair_tracker().next_repair_command();
     rlogger.info("repair[{}]: starting user-requested repair for keyspace {}, repair id {}, options {}", id.uuid, keyspace, id.id, options_map);
 
+    task_status.id = id.uuid;
+    task_status.state = task_manager::task_state::running;
```

Comment:
setting the task state can be done by the task constructor.
it's ok if we have an `uninitialized` default state (= 0) for default constructed tasks.
It may make debuggin easier.